src/app/plugins/broadcast.plugin.ts

```diff
@@ -0,0 +1,71 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, ClassType, Maybe } from '../../common/types';
+import Constants from '../../common/constants';
+import { RichEmbed, TextChannel } from 'discord.js';
+
+export class BroadcastPlugin extends Plugin {
+  public name: string = 'Broadcast';
+  public description: string = 'Sends an annoucement to all class channels';
+  public usage: string = '!broadcast <message>';
+  public pluginAlias = [];
+  public permission: ChannelType = ChannelType.Admin;
+  public pluginChannelName: string = Constants.Channels.Staff.AdminChat;
```

Comment:
should be necessary https://github.com/joey-colon/lion/blob/5d686fd2ead1da4121b2e2348e5401c5da0d683d/src/app/plugins/addclasschans.plugin.ts#L15